rules/linux/process_creation/proc_creation_lnx_device_driver_insert_remove.yml

```diff
@@ -0,0 +1,28 @@
+title: Detect Adding Kernel Modules - Linux
+id: 85b78882-a58d-405f-bbb3-201d41812e8c
+status: test
+description: Adversaries may attempt to insert/remove rootkits onto a victim host. This rule detects the use of commands such as 'insmod', 'modprobe', and 'rmmod' which are commonly used to load or remove kernel modules.
+references:
+    - https://attack.mitre.org/techniques/T1014/
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1014/T1014.md
```

Comment:
Permalinks needs to be used, and additional references are required

Reply:
Hi,
Could you tell me which permalinks are required?

Reply:
https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files